modules/currency_denomination/commerce_pos_currency_denomination.module

```diff
@@ -0,0 +1,195 @@
+    'title' => 'Delete Currency Denomination Type',
+    'page callback' => 'drupal_get_form',
+    'type' => MENU_CALLBACK,
+    'page arguments' => array('commerce_pos_currency_delete_denomination_type_form', 5),
+    'access callback' => '_commerce_pos_access_check',
+    'access arguments' => array('process commerce pos sales'),
+    'file' => 'includes/commerce_pos_currency_denomination.admin.inc',
+  );
+
+  $items['admin/commerce/config/pos/denomination-type/%/denomination/add'] = array(
+    'title' => 'Add Denomination',
+    'page callback' => 'drupal_get_form',
+    'type' => MENU_CALLBACK,
+    'page arguments' => array('commerce_pos_currency_add_denomination_form', 5),
+    'access callback' => '_commerce_pos_access_check',
+    'access arguments' => array('process commerce pos sales'),
+    'file' => 'includes/commerce_pos_currency_denomination.admin.inc',
+  );
+
+  $items['admin/commerce/config/pos/denomination-type/%/denomination/view'] = array(
+    'title' => 'View Denominations',
+    'page callback' => 'commerce_pos_currency_view_denominations',
+    'type' => MENU_CALLBACK,
+    'page arguments' => array(5),
+    'access callback' => '_commerce_pos_access_check',
+    'access arguments' => array('process commerce pos sales'),
+    'file' => 'includes/commerce_pos_currency_denomination.admin.inc',
+  );
+
+  $items['admin/commerce/config/pos/denomination-type/%/denomination/%/delete'] = array(
+    'title' => 'Delete Denomination',
+    'page callback' => 'drupal_get_form',
+    'type' => MENU_CALLBACK,
+    'page arguments' => array('commerce_pos_currency_delete_denomination_form', 5, 7),
+    'access callback' => '_commerce_pos_access_check',
+    'access arguments' => array('process commerce pos sales'),
+    'file' => 'includes/commerce_pos_currency_denomination.admin.inc',
+  );
+
+  return $items;
+}
+
+/**
+ * Returns all the denominations from the db for a currency code.
+ *
+ * @param array $currency_codes
+ * @return array
+ */
+function commerce_pos_currency_denomination_get_denominations($currency_codes = array()) {
+  $denominations = array();
+
+  // Let's first grab all the denominations under the given currency codes.
+  $query = db_select('commerce_pos_currency_denomination', 'd');
+  $query->fields('d');
+
+  if ($currency_codes) {
+    $currency_ids = commerce_pos_denomination_get_currency_id($currency_codes);
+    $query = $query->condition('currency_id', $currency_ids, 'IN');
+  }
+  $results = $query->execute();
+
+  // Let's organize them by currency id and then, by denomination id.
+  foreach ($results as $result) {
+    $denominations[$result->currency_id][$result->denomination_id] = $result;
+  }
+
+  return $denominations;
+}
+
+/**
+ * Returns the configured currency denomination types.
+ *
+ * @param array $currency_codes
+ * @return array
+ */
+function commerce_pos_denomination_get_currency_types($currency_codes = array()) {

```

Comment:
All functions provided by a module should be prefixed with the module name, in this case 'commerce_pos_currency_denomination', to avoid naming conflicts. There are a few other functions with this same issue.